_data/accessibility-tests/icon-list.yml

```diff
@@ -0,0 +1,48 @@
+title: Icon list
+component_status: pass
+test_items:
+# General tests
+  - summary: Descriptive alt text identifies icon purpose.
+    summary_additional: When you view icons in an icon list component, you can mouse over an icon to see its meaning or purpose. Decorative icons will have no alt text.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.1.1
+  - summary: Color alone is not used to convey information.
+    summary_additional: When you view an icon list, you never have to rely on color alone to understand the meaning of the icons being used.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.1
+  - summary: Icons meet color contrast requirements.
+    summary_additional: When you look at an icon list and use ANDI or another color contrast analyzer to look at the hex numbers, contrast between the icons and the background is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.11
+  - summary: Icon list content is written in plain language.
+    summary_additional: When you read the content in the icon list, the information is easy to understand. Content is written at the appropriate level for the audience.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.1.5
+  - summary: Icons used in an icon list are used the same way across the site.
+    summary_additional: When you see or tab to any icon list, icons used will always be represent the same thing in other lists on other parts of the website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.2.4
+# Zoom/screen magnification tests
+  - summary: Text and icons retain functionality when screen is magnified.
+    summary_additional: When you zoom in to 200%, icons and text are readable and retain all of their functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.4
+# Screen reader tests
+  - summary: Screen reader announces icon meaning and purpose.
+    summary_additional: When you listen to icon descriptions in a list using a screen reader, you can understand their meaning. Alt text descriptions indicate purpose.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.6
```

Comment:
**Question:** Are we moving forward with this test? We ended up closing out uswds/uswds#6194 which aimed to add `title` (alt text) to the icons. Should we remove this test?

Alternatively, should we update it to say _if_ icons are meaningful, this test is relevant?

CC: @alex-hull @amyleadem

Reply:
I made a similar comment somewhere here, that I feel like we should keep this test and mark it as conditional. I feel like if they use a meaningful icon, but this test is removed, then it may discourage people from using this test. But I this the language change of "if" should be used!

Reply:
In that case, we should add a `status_details:` key and maybe echo some of the language in the guidance:

> If you wish to expose icons to screen readers:
> - Provide descriptive text for each icon.
> - Remove the aria-hidden="true" attribute and add a aria-labelledby attribute with a value that matches the id of a title element added inside the SVG.

**suggestion:** Maybe something like

> This test is only relevant if you choose to expose icons to screen readers. Visit input list accessibility guidance for more information

This would appear here:

![image](https://github.com/user-attachments/assets/37460a50-6369-4abb-aaa0-e2554c242b47)

Is that too much? Should the "if" be in the test summary instead so users see it sooner?

@alex-hull  @amyleadem  @finekatie

Reply:
I think this works! Definitely not too much. I think the if is fine in the status message. I think usually the conditional test status are usually in an "if" scenario. And should it be "icon list" and not "input list" in the test status?

Reply:
Yep! Typo on my end!

**Suggestion:** Can you try something like this @RachelCorsino ?

```suggestion
# Screen reader tests
  - summary: Screen reader announces icon meaning and purpose.
    summary_additional: When you listen to icon descriptions in a list using a screen reader, you can understand their meaning. Alt text descriptions indicate purpose.
    
    test_status: conditional
    test_type: screen_reader
    status_details: This test is only relevant if you choose to expose icons to screen readers. Visit icon list accessibility guidance for more information.
    version_tested: 3.8.2
    wcag_criterion: 1.3.6
```

Reply:
**suggestion:** Tweaked @mahoneycm's recommendation a bit to:
- Signal in the criteria summary that this doesn't always apply (added "when appropriate")
- Signal that there is a reason why users should choose to expose these to screen readers
- Add a link to the guidance section 

```suggestion
# Screen reader tests
  - summary: Screen reader announces icon meaning and purpose when appropriate.
    summary_additional: When you listen to icon descriptions in a list using a screen reader, you can understand their meaning. Alt text descriptions indicate purpose.
    test_status: conditional
    test_type: screen_reader
    status_details: This test is only relevant if your icons have meaning and you choose to expose them to screen readers. Visit <a href="https://designsystem.digital.gov/components/icon-list/#accessibility-icon-list">icon list accessibility guidance</a> for more information.
    version_tested: 3.8.2
    wcag_criterion: 1.3.6
```

Reply:
Resolved in [cc22788](https://github.com/uswds/uswds-site/pull/3008/commits/cc22788af5574655de147eb2a3eb153a29db5ab5)!